### ISSUE TITLE
[Bug]  V2.0 preview after hibernate bar is empty (Win11 x64)

### ISSUE THREAD

**Issue by jawee** [glzr-io/zebar#109]
### Describe the bug

When I wake my computer up from hibernate, the bar is completely empty. Reloading the config a couple of times solves it.

It shows this error in the developer console. Line 61 from https://github.com/glzr-io/zebar/blob/main/packages/client-api/src/providers/weather/create-weather-provider.ts

### Stack trace or error logs (if applicable)

```shell
Uncaught TypeError: Cannot read properties of null (reading 'approxLongitude')
    at xr (index.js:653:67)
    at async index.js:773:20
    at async Promise.all (index 6)
    at async Lr (index.js:771:27)
    at async Object.Hr [as createProviderGroup] (index.js:723:23)
    at async <anonymous>:5:17
```

### Version number

2.0-preview

**Comment by khusmann** [comment]
This happened with v2.0-preview for me, but seems to be fixed in 2.1.1 (just released)

**Comment by lars-berger** [comment]
Thanks for reporting this 🙏 Fixed in the latest version
